python/ray/serve/common.py

```diff
@@ -49,24 +48,28 @@ class ReplicaName:
     replica_suffix: str
     replica_tag: ReplicaTag = ""
     delimiter: str = "#"
+    prefix: str = "SERVE_REPLICA::"
 
     def __init__(self, deployment_tag: str, replica_suffix: str):
         self.deployment_tag = deployment_tag
         self.replica_suffix = replica_suffix
         self.replica_tag = f"{deployment_tag}{self.delimiter}{replica_suffix}"
 
+    @staticmethod
+    def is_replica_name(actor_name: str) -> bool:
+        return (actor_name.startswith(ReplicaName.prefix)
+                and actor_name.count("#") == 1)
```

Comment:
what's the `#` check for? seems potentially dangerous (if there's `#` in deployment name for example)?